components/slider/Slider.razor.cs

```diff
@@ -477,6 +463,7 @@ protected override void OnParametersSet()
             ClassMapper.Clear()
                 .Add(PreFixCls)
                 .If($"{PreFixCls}-disabled", () => Disabled)
+                .If($"{PreFixCls}-horizontal", () => !Vertical)
```

Comment:
Added this class because it is in the react version whenever it is horizontal. Didn't appear to affect any visuals for now though. Perhaps a style update will make use of it in the future and prevent a bug.

Reply:
Our current style is only synchronized to 4.20.7 (and many changes to the component html are not synchronized), and the latest one needs to check whether the component style is updated before merging. Could you please take a look at it?

https://github.com/ant-design-blazor/ant-design-blazor/pull/2814